spark/src/main/java/org/apache/iceberg/spark/SparkReadOptions.java

```diff
@@ -50,4 +50,10 @@ private SparkReadOptions() {
 
   // Set ID that is used to fetch file scan tasks
   public static final String FILE_SCAN_TASK_SET_ID = "file-scan-task-set-id";
+
+  // skip snapshots of type delete while reading stream out of iceberg table
+  public static final String READ_STREAM_SKIP_DELETE = "read-stream-skip-delete";
```

Comment:
on a second thought this naming seems like a better one - `skip-deletes-on-stream-read`
my problem with the older one was that - `skipping deletes` is not popping out - `read-stream` is.